humanoid_league_team_communication/scripts/TeamCommTest.py

```diff
@@ -1,45 +1,43 @@
 #!/usr/bin/env python3
```

Comment:
The file is named `TeamCommTest.py` anyone against changing it to `team_comm_test.py`

Reply:
Nope